scripts/protoDC2/generateDc2InstCat.py

```diff
@@ -12,20 +13,41 @@
 
 class MaskedPhoSimCatalogPoint(PhoSimCatalogPoint):
 
+    disable_proper_motion = False
+
     min_mag = None
 
     column_outputs = ['prefix', 'uniqueId', 'raPhoSim', 'decPhoSim', 'maskedMagNorm', 'sedFilepath',
-                      'redshift', 'gamma1', 'gamma2', 'kappa', 'raOffset', 'decOffset',
+                      'redshift', 'shear1', 'shear2', 'kappa', 'raOffset', 'decOffset',
```

Comment:
I changed `shear1` and `shear2` to `gamma1` and `gamma2` on master last week.  I can issue a new release this week,since I think both conventions are now floating around.

Reply:
@danielsf I'm about to make a new branch with this change, the `angularSeparation` fix, and (presumably) a change to make the disabling of the application of proper motion as the default behavior of the `generateDc2InstCat.py` script.   Can you let me know when you've issued the sims release with these changes?

Reply:
Okay.  I will start that today.